src/ballet/shred/fd_shred.h

```diff
@@ -155,7 +162,7 @@ struct __attribute__((packed)) fd_shred {
      For legacy type shreds, the low four bits are set to static patterns.
      For merkle type shreds, the low four bits are set to the number of non-root nodes in the inclusion proof.
      For chained merkle code type shreds, the 3rd highest bit represents if the merkle tree is chained.
-     For chained merkle data type shreds, the 4th highest bit represents if the merkle tree is 
+     For chained merkle data type shreds, the 4th highest bit represents if the merkle tree is
```

Comment:
Looks like you forgot to update this?

Reply:
Actually, it was my editor trimming whitespace but I updated it anyhow